### DISCUSSION TITLE
Plots: Bulk for sale

### DISCUSSION THREAD

**Discussion by Misquoth** [TownyAdvanced/Towny#4838] [Q&A] [answered]
for towns with many plots, is there a way to set prices in bulk as opposed to walking to each and every one?

**Comment by LlmDl** [comment, marked as answer]
Using the plot groups is an option here. See the how towny works wikipage.

**Reply by Misquoth** [reply]
Yes, I've read that. I'm not meaning to sell, say 9 plots as 1 plot.

I'm looking to say: Town x has 30 plots. Is there a way to set all plots in a town for sale at the same price? 

The plots would be sold individually. 

What I'm trying to avoid is walking to each plot, one by one, and having to run the same command over and over again.

**Reply by Misquoth** [reply]
Or also, a way to set all NFS plots to for sale. 

I have a couple players who have already claimed many many plots and they are asking if there is a way for them to avoid modifying each one.

Thanks

**Reply by LlmDl** [reply]
`/plot fs AMOUNT {circle | rect} X` where X is the radius of the area you want to put up for sale.

**Reply by LlmDl** [reply]
https://github.com/TownyAdvanced/Towny/wiki/Towny-Commands#plot

**Reply by Misquoth** [reply]
THANKS!!
